app/helpers/application_helper.rb

```diff
@@ -3,6 +3,14 @@ def date_format(date)
     date.in_time_zone.strftime "%B %Y"
   end
 
+  def date_get_month(date)
```

Comment:
⚠️ Since this is logic for the view, I felt it was best to put it in the `application_helper` file.